srcpkgs/krakenx/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'krakenx'
+pkgname=krakenx
+version=0.0.3
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-pip"
+makedepends="python3"
```

Comment:
I don't think `python3` is needed in the `makedepends`

Reply:
i missed libusb, thanks for pointing that out.
On https://github.com/KsenijaS/krakenx libusb-32bit is specifically mentioned, is it necessary to declare 32 bit in the dependencies or better to omit it?